### PR TITLE
add edit link

### PR DESCRIPTION
**Pull request by forresto**
This makes it easier to make threads / quote yourself, by separating the edit link from the timestamp.

### CODE REVIEW

File: scripts/entry.js [hunk 1]
```diff
@@ -113,14 +113,17 @@ function Entry(data,host)
...
     var operation = '';
-    if(this.host.json.name == r.home.portal.json.name)
-      operation = 'edit:'+this.id+' '+this.message.replace(/\'/g,"&apos;");
-    else if(this.whisper)
+    if(this.whisper)
       operation = "whisper:"+this.host.json.name+" ";
     else
       operation = "quote:"+this.host.json.name+"-"+this.id+" ";
 
     html += this.editstamp ? "<c class='editstamp' data-operation='"+r.escape_attr(operation)+"' title='"+this.localtime()+"'>edited "+timeSince(this.editstamp)+" ago</c>" : "<c class='timestamp' data-operation='"+operation+"' title='"+this.localtime()+"'>"+timeSince(this.timestamp)+" ago</c>";
```
Comment (forresto) [inline scripts/entry.js]: I don't understand why one has `r.escape_attr` and the other doesn't here.

**Comment by neauoire** [comment]
They should both have it I think

File: scripts/entry.js (hunk 1 above)
Comment (composition4) [reply scripts/entry.js]: Looks like a modal. it's checking for `this.editstamp`

**Comment by composition4** [comment]
image

#137 or #139 is causing this

**Comment by forresto** [comment]
@composition4 :man_facepalming: #139 should fix that.

**Pull request merged**